collections/_evergreen/linux-desktop.html

```diff
@@ -0,0 +1,191 @@
+---
+layout: evergreen
+title: Linux Desktop
+mathjax: true
+description: "Linux distributions are commonly recommended for user freedom and privacy. This page will go over distribution selection and general configuration for privacy and security. This is not a guide on how to use Linux."
+---
+
+<h2 id="beginner-friendly" class="anchor">
+    <a href="#traditional"><i class="fas fa-link anchor-icon"></i></a>
+    Traditional distributions
+</h2>
+
+<p>These distributions are recommended on the basis that they ship close to vanilla packages and stay very close to upstream, allowing them to get security features/bug fixes very quickly.</p>
+
+{% for item_hash in site.data.operating-systems.linux-desktop %}
+{% assign item = item_hash[1] %}
+
+{% if item.type == "Recommendation" %}
+{% include recommendation-card.html %}
+{% endif %}
+
+{% endfor %}
+
+<h2 id="immutable" class="anchor">
+    <a href="#immutable"><i class="fas fa-link anchor-icon"></i></a>
+    Immutable distributions
+</h2>
+
+<p>These distributions are recommended as they have <a href="#atomic">atomic</a> updates, allowing users to update their systems very liberally without fear of instability and breakage.</p>
+
+{% for item_hash in site.data.operating-systems.linux-desktop-immutable %}
+{% assign item = item_hash[1] %}
+
+{% if item.type == "Recommendation" %}
+{% include recommendation-card.html %}
+{% endif %}
+
+{% endfor %}
+
+<h2 id="tor" class="anchor">
+    <a href="#tor"><i class="fas fa-link anchor-icon"></i></a>
+    Tor focused distributions
+</h2>
+
+<p>These distributions are recommended as they focus on providing anonimity for the user.</p>
+
+{% for item_hash in site.data.operating-systems.linux-desktop-tor %}
+{% assign item = item_hash[1] %}
+
+{% if item.type == "Recommendation" %}
+{% include recommendation-card.html %}
+{% endif %}
+
+{% endfor %}
```

Comment:
I think I would swap the order here and do:

1. Traditional
2. Tor-focused
3. Immutable/Atomic

```suggestion
<h2 id="tor" class="anchor">
    <a href="#tor"><i class="fas fa-link anchor-icon"></i></a>
    Tor-focused distributions
</h2>

<p>These distributions are recommended because they focus on providing maximum anonymity using the Tor network.</p>

{% for item_hash in site.data.operating-systems.linux-desktop-tor %}
{% assign item = item_hash[1] %}

{% if item.type == "Recommendation" %}
{% include recommendation-card.html %}
{% endif %}

{% endfor %}

<h2 id="immutable" class="anchor">
    <a href="#immutable"><i class="fas fa-link anchor-icon"></i></a>
    Immutable distributions
</h2>

<p>These distributions are recommended because they are <a href="#atomic">atomic</a>, meaning users can update and modify their systems without fear of instability and breakage.</p>

{% for item_hash in site.data.operating-systems.linux-desktop-immutable %}
{% assign item = item_hash[1] %}

{% if item.type == "Recommendation" %}
{% include recommendation-card.html %}
{% endif %}

{% endfor %}
```

Reply:
I think the current ordering makes more sense because the Tor Focused distributions are a lot different from the ones at the top